tests/Unit/Evolution/Particles/MonteCarlo/Test_EvolvePackets.cpp

```diff
@@ -94,10 +94,23 @@ SPECTRE_TEST_CASE("Unit.Evolution.Particles.MonteCarloEvolution",
       std::array<DataVector, 2>{{zero_dv, zero_dv}},
       std::array<DataVector, 2>{{zero_dv, zero_dv}}};
 
+  // Set non-zero value that should nearly never lead
+  // to interaction, to get non-zero interaction terms.
```

Comment:
I'll update the comment after code review -- but in fact, due to the use of a minimum value of 1.e-100 in the RNG when determining the time to the next event, we are explicitly guaranteed that no scattering / absorption will happen.